trusted_os/rpc.go

```diff
@@ -176,8 +176,12 @@ func (r *RPC) ReadRPMB(buf []byte, n *uint32) error {
 //
 // The diversifier is AES-CBC encrypted using the internal OTPMK key.
 func (r *RPC) DeriveKey(diversifier [aes.BlockSize]byte, key *[sha256.Size]byte) (err error) {
-	if !imx6ul.SNVS.Available() && !debug {
-		return errors.New("Weird - SNVS not available but we're not in debug?!")
+	if debug && !imx6ul.Native {
+		if imx6ul.Native {
```

Comment:
The logic here doesn't seem quite right - `if ... && !Native` --> the nested `if Native` can never be true? 

did you mean to strip off the `Native` bit on the outer `if`?

Reply:
Typo, sorry for that, should be fixed in https://github.com/transparency-dev/armored-witness-os/pull/219/commits/0e90523aed6c64b0b2f3bb77b0bed2d4350f9283